crdt_lib/include/list.h

```diff
@@ -0,0 +1,178 @@
+//
+// Created by sumeet on 11/18/20.
+//
+
+#ifndef NUPAD_LIST_H
+#define NUPAD_LIST_H
+
+#include <list>
+#include <utility>
+#include <glog/logging.h>
+
+#include "context.h"
+#include "operation.h"
+
+namespace nupad::crdt {
+
+    /* All definitions are in-place to avoid linker error, this is a standard practice
+     * used in STL.
+     * ref: https://stackoverflow.com/questions/8752837/undefined-reference-to-template-class-constructor
+     */
+    template<typename T>
+    class CRDTList {
```

Comment:
sumeet says: Rename to `DoublyLinkedList`

Reply:
Done